spec/swagger/custom_properties_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+module Swagger
+  module V2
+    describe API do
+      let(:swagger_file) { 'spec/fixtures/custom-properties.yaml' }
+      let(:swagger) { Swagger.load swagger_file }
+
+      context 'custom properties' do
+        describe 'getting a custom property' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.